projects/ui/src/components/Nav/FolderMenu.tsx

```diff
@@ -139,11 +176,24 @@ const FolderMenu: FC<{
       <Popper
         open={popoverOpen}
         anchorEl={anchorEl}
-        placement="bottom-start"
+        placement={popoverPlacement || 'bottom-start'}
         disablePortal
         sx={{
-          zIndex,
+          zIndex: zIndex,
+          visibility: mobileWindow ? 'hidden' : 'visible',
         }}
+        modifiers={[
+          {
+            name: 'computeStyles',
+            options: {
+              gpuAcceleration: false,
```

Comment:
Fine to leave this, but curious for my own learning if there was any particular reason